controlled_navigation/controlled_navigation.py

```diff
@@ -32,18 +38,70 @@ class XBlockControlledNavigation(
         default=_("Content with Controlled Navegation"),
     )
 
+    randomness = Boolean(
+        display_name=_("Randomness"),
+        help=_("When enabled, the children will be displayed in a random order."),
+        scope=Scope.settings,
+        default=False,
+    )
+
+    next_button_text = String(
+        display_name=_("Next Button Text"),
+        help=_("Text for the next button."),
```

Comment:
```suggestion
        help=_("Text for the next button used to navigate forward through the components' children."),
```
Or something like it?

Reply:
Same comment for the previous field.

Reply:
Done: https://github.com/eduNEXT/xblock-controlled-navigation/commit/3816dcf97e04141542f99d2a3f73b967e7e9ddd5